src/components/ShippingDetails/ShippingDetails.js

```diff
@@ -1,43 +1,38 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import styled from 'styled-components';
 
+import { colorLightGrey } from '../../styles/designTokens';
+import { formatDate } from '../../utils';
 import Shipping from '../Icons/Shipping';
 import Text from '../Text';
-import { colorLightGrey } from '../../styles/designTokens';
-
-const Wrapper = styled.div`
-  display: flex;
-  align-items: center;
-`;
-
-const Message = styled.div`
-  flex-grow: 1;
-  padding: 0 1rem;
-`;
+import ShippingText from './ShippingText';
+import Wrapper from './Wrapper';
 
-const ArrivalDates = styled.span`
-  font-weight: bold;
-`;
+function ShippingDetails() {
+  const getArrivalDates = () => {
+    const startDate = new Date();
+    const endDate = new Date();
+    endDate.setDate(endDate.getDate() + 3);
 
-const ShippingDetails = ({ minDate, maxDate, shippingPromo }) => (
-  <Wrapper>
-    <Shipping fill={colorLightGrey} />
-    <Message>
-      <Text>
-        Ship to Me <br />
-        <span>Arrives </span>
-        <ArrivalDates>{`${minDate} - ${maxDate}`}</ArrivalDates>
-      </Text>
-      {shippingPromo && <Text>{shippingPromo}</Text>}
-    </Message>
-  </Wrapper>
-);
+    return `${formatDate(startDate)} - ${formatDate(endDate)}`;
+  };
 
-ShippingDetails.propTypes = {
-  minDate: PropTypes.string.isRequired,
-  maxDate: PropTypes.string.isRequired,
-  shippingPromo: PropTypes.string
-};
+  return (
+    <>
+      <Text bold>Item can only be shipped Standard</Text>
+      <Wrapper>
+        <Shipping fill={colorLightGrey} />
+        <div>
+          <ShippingText>Arrives</ShippingText>
```

Comment:
Well it is a bit complicated as for only clearing paddings and margins. What about creating a single component, with raw text and spans, and handle that by styling? 
```
<ShippingInfo>
Arrives
Ship to Me<span>{getArrivalDates()}</span>
</ ShippingInfo>
```